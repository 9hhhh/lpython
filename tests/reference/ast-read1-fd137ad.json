{
    "basename": "ast-read1-fd137ad",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/read1.f90",
    "infile_hash": "4f1b270c03dbd1104dae11dfcf0f57cf762c2534ed38f8fcf16a54a1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-read1-fd137ad.stdout",
    "stdout_hash": "1f47df4312cc768c33a5280706ccd8f5d9b12e2478e2cdcea8a8d964",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}