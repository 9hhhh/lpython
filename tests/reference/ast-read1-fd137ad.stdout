(TranslationUnit [(Print () [a b]) (Print () [b]) (Print () [hour]) (Print () [b]) (Print () [b c e])])
